utils/Web3Utils.ts

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
This seems to be a good candidate to put in the utils repo? where did this come from?

Reply:
we'be been using it in both sides but its based off of the web3-utils package